### PR TITLE
17555 error message displayed after wp edit

### PR DESCRIPTION
**Pull request by hschink**
[`* `#17555` Error message displayed after work package edit`](https://community.openproject.org/work_packages/17555)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://hotfix-17555-error-message-displayed-after-wp-edit.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/work_packages/controllers/work-packages-list-controller.js [hunk 1]
```diff
@@ -205,19 +205,8 @@ angular.module('openproject.workPackages.controllers')
...
-      var queryString = UrlParamsHelper.encodeQueryJsonParams($scope.query);
-      relativeUri += (($scope.query.isNew()) ? '?' : '&') + 'query_props=' + queryString;
-    }
-
-    return relativeUri;
-  }
-
   $scope.maintainBackUrl = function() {
-    $scope.backUrl = getCurrentStateUrl();
+    $scope.backUrl = $location.url();
```
Comment (myabc) [inline app/assets/javascripts/angular/work_packages/controllers/work-packages-list-controller.js:209]: :+1:

**Comment by ulferts** [comment]
When clicking on "Edit" on the work package pane, a js error is produced. `$location` seems to not be injected in the work-package-details-toolbar-directive

**Comment by hschink** [comment]
@ulferts 5529c2c addresses the issue https://github.com/opf/openproject/pull/2269#issuecomment-65642374.

**Comment by ulferts** [comment]
Free to merge once travis agrees.

**Comment by ulferts** [comment]
@hschink I have not yet merged release/4.0 back into dev for that. Can you do that please?

**Pull request merged**